### ISSUE TITLE
Renaming ansible.egg-info fails when sourcing hacking/env-setup

### ISSUE THREAD

**Issue by wincent** [ansible/ansible#10598]
##### Issue type

Bug report
##### Ansible version

Git checkout of 1.9.0.1-1
##### Environment

I've tested this on OS X 10.10.2. This is using the system Python at `/usr/bin/python`, which reports itself as version 2.7.6.
##### Summary

When I source `hacking/env-setup` [this attemped `mv`](https://github.com/ansible/ansible/blob/v1.9.0.1-1/hacking/env-setup#L48) is blowing up with:

```
mv: rename ansible.egg-info to /Users/glh/code/wincent/vendor/ansible/lib/ansible.egg-info: No such file or directory
```
##### Steps to reproduce

```
source hacking/env-setup
```
##### Expected results

It completes without errors
##### Actual results

Full output:

```
running egg_info
creating lib/ansible.egg-info
writing requirements to lib/ansible.egg-info/requires.txt
writing lib/ansible.egg-info/PKG-INFO
writing top-level names to lib/ansible.egg-info/top_level.txt
writing dependency_links to lib/ansible.egg-info/dependency_links.txt
writing manifest file 'lib/ansible.egg-info/SOURCES.txt'
reading manifest file 'lib/ansible.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
no previously-included directories found matching 'v2'
no previously-included directories found matching 'docsite'
no previously-included directories found matching 'ticket_stubs'
no previously-included directories found matching 'packaging'
no previously-included directories found matching 'test'
no previously-included directories found matching 'hacking'
no previously-included directories found matching 'lib/ansible/modules/core/.git'
no previously-included directories found matching 'lib/ansible/modules/extras/.git'
writing manifest file 'lib/ansible.egg-info/SOURCES.txt'
mv: rename ansible.egg-info to /Users/glh/code/wincent/vendor/ansible/lib/ansible.egg-info: No such file or directory

Setting up Ansible to run out of checkout...

PATH=/Users/glh/code/wincent/vendor/ansible/bin:/Users/glh/.rbenv/shims:/Users/glh/bin:/Users/glh/.shells/bin:/usr/local/bin:/usr/local/sbin:/opt/facebook/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/opt/X11/bin:/Users/glh/devtools/buck/bin:/usr/local/git/bin:/usr/local/munki:/Users/glh/.rbenv/shims:/Users/glh/bin:/Users/glh/.shells/bin:/usr/local/sbin:/Users/glh/bin/ec2-api-tools/bin:/Users/glh/src/devtools/arcanist/bin:/Users/glh/bin/ec2-api-tools/bin:/Users/glh/src/devtools/arcanist/bin
PYTHONPATH=/Users/glh/code/wincent/vendor/ansible/lib:
MANPATH=/Users/glh/code/wincent/vendor/ansible/docs/man:

Remember, you may wish to specify your host file with -i

Done!
```

The error appears to be harmless, as the script continues executing (unless you have `set -e`) and Ansible still seems to work afterwards, but it would be nice to fix so that you _can_ source the script in an environment which uses `set -e`.

**Comment by sivel** [comment]
This was already resolved in https://github.com/ansible/ansible/pull/10563

**Comment by bcoca** [comment]
closing as dupe, also already solved  in #10563 as mentioned above
